### PR TITLE
#4 OAuth with Devise

### PR DESCRIPTION
**Pull request by christinaforney**

### CODE REVIEW

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+module Users
+  class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+    def google_oauth2
+      @user = User.find_for_google_oauth2(request.env["omniauth.auth"],
+                                          current_user)
+
+      if @user.persisted?
+        flash[:notice] = I18n.t "devise.omniauth_callbacks.success",
+          kind: "Google"
```
Comment (andrewpurcell) [inline app/controllers/users/omniauth_callbacks_controller.rb]: fu hound.

File: app/controllers/users/confirmations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```
Comment (andrewpurcell) [inline app/controllers/users/confirmations_controller.rb]: delete this class

File: app/controllers/users/unlocks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Users::UnlocksController < Devise::UnlocksController
```
Comment (andrewpurcell) [inline app/controllers/users/unlocks_controller.rb]: delete

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (andrewpurcell) [inline app/models/user.rb]: don't need

**Comment by christinaforney** [comment]
Closes #4

**Pull request merged**